interface-definitions/interfaces-pppoe.xml.in

```diff
@@ -50,6 +50,19 @@
               <constraintErrorMessage>Host-uniq must be specified as hex-adecimal byte-string (even number of HEX characters)</constraintErrorMessage>
             </properties>
           </leafNode>
+          <leafNode name="holdoff">
+            <properties>
+              <help>Delay before re-dial to the access concentrator when PPP session terminated by peer (in seconds)</help>
+              <valueHelp>
+                <format>u32:0-86400</format>
```

Comment:
Holdoff must be in the range 1 to 86400, but there is `0-86400`

Reply:
Fixed.